packages/editor/src/components/entities-saved-states/entity-type-list.js

```diff
@@ -6,20 +6,35 @@ import { some } from 'lodash';
 /**
  * WordPress dependencies
  */
+import { __, _n } from '@wordpress/i18n';
 import { useSelect } from '@wordpress/data';
-import { PanelBody } from '@wordpress/components';
-import { page, layout } from '@wordpress/icons';
+import { PanelBody, PanelRow } from '@wordpress/components';
 import { store as coreStore } from '@wordpress/core-data';
 
 /**
  * Internal dependencies
  */
 import EntityRecordItem from './entity-record-item';
 
-const ENTITY_NAME_ICONS = {
-	site: layout,
-	page,
-};
+function getEntityDescription( entity, length ) {
+	switch ( entity ) {
+		case 'site':
+			return _n(
+				'This change will affect your whole site.',
+				'These changes will affect your whole site.',
+				length
+			);
+		case 'wp_template':
+			return _n(
+				'This change will affect pages and posts that use this template.',
+				'These changes will affect pages and posts that use these templates.',
+				length
+			);
+		case 'page':
+		case 'post':
+			return __( 'The following content has been modified.' );
```

Comment:
Should we move these descriptions to the entities config instead?

Reply:
You mean https://github.com/WordPress/gutenberg/blob/ba5f6d061934625027beab9d2bf600ae42a63be9/packages/core-data/src/entities.js? Yeah, that might make sense 👍